src/cpp/r/R/Api.R

```diff
@@ -841,6 +841,33 @@ options(terminal.manager = list(terminalActivate = .rs.api.terminalActivate,
    .Call("rs_systemUsername", PACKAGE = "(embedding)")
 })
 
+# store callback functions to be executed after a specified chunk
+# and return a handle to unregister the chunk
+.rs.addApiFunction("registerChunkCallback", function(chunkCallback = function(chunkName, chunkCode){}) {
```

Comment:
I don't think this should have a default value since you'd never really want to register an no-op function as a callback.